src/main/java/com/softserve/ita/java442/cityDonut/constant/ErrorMessage.java

```diff
@@ -4,4 +4,10 @@
     public static final String USER_NOT_FOUND_BY_ID = "The user does not exist by this id: ";
     public static final String PROJECT_NOT_FOUND_BY_ID = "The project does not exist by this id";
     public static final String INCORRECT_USER_PASSWORD = "The user`s password is incorrect: ";
+
```

Comment:
delete row

Reply:
It's more convenient to read messages by blocks. For example, I have in block down all messages needed for files, so if I need to add a new one, I will check all messages in this block and then decide what to do. If all messages are in one block, I will have to read many of them. If you whant, you can check code convention of this file in GreenCity